src/yb/master/master_main.cc

```diff
@@ -69,6 +71,16 @@ static int MasterMain(int argc, char** argv) {
   // Reset some default values before parsing gflags.
   FLAGS_rpc_bind_addresses = strings::Substitute("0.0.0.0:$0", kMasterDefaultPort);
   FLAGS_webserver_port = kMasterDefaultWebPort;
+
+
+  auto host_name = GetHostname();
+  if (host_name.ok()) {
+    FLAGS_metric_node_name = (*host_name) + ":" + std::to_string(FLAGS_webserver_port);
```

Comment:
Instead of webserver_port, we should use kMasterDefaultPort.

Reply:
I have made the required changes, however, I am not able to test the output. I'll reach out on Slack.